error-handling/src/main/java/com/palantir/remoting1/errors/SerializableError.java

```diff
@@ -37,36 +35,28 @@
 
     public abstract String getMessage();
 
-    @JsonProperty("exceptionClass")
```

Comment:
changing this breaks wire format right? eg: if servers pick up this change then old clients won't be able to deserialize because the property name has changed.